### ISSUE TITLE
Feature: Unmark button in the visgroups window

### ISSUE THREAD

**Issue by Konclan** [ChaosInitiative/Portal-2-Community-Edition#492]
## Which component should be improved?
Hammer visgroups window

## Describe your suggestion
The visgroups window has a mark button to add selected objects to a selected visgroup. There should be an unmark button to do the opposite and remove selected objects from a selected visgroup

## Expected result
When having objects and a visgroup selected, pressing the unmark button should remove the objects from the selected visgroup.

**Comment by Luke18033** [comment]
That's not what the mark button does, it selects objects which are already in the visgroup. An unmark button could maybe be useful in certain situations, but generally wouldn't be needed.

**Comment by Konclan** [comment]
Yeah I just realized that, I was editing when you commented lol. I have a lot of notes I have written down that I'm currently going through and I just mixed up how the mark button actually works. I would still like an unmark button.
